File: Unidad_4/Programa1_U4.cs
using System;

namespace MyCompany.BaseDeDatos
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] edades = new int[] { 17, 19, 18, 18, 18, };
            int[] edades2 = new int [] { 17, 19, 18, 18, 18 };

            for (int i = 0; i < edades.Length; i++)
            {
                Console.WriteLine("indice: " + i + "es: " + edades[i]);
                if (edades[i] >= 18)
                    Console.WriteLine("Eres Mayor");
                else
                    Console.WriteLine("Eres Menor");
            }

        }
    }
}

File: Unidad_4/Java/Programa1_U4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa1_u4;

/**
 *
 * @author leebc
 */
public class Programa1_U4 {

    public static void main(String[] args) {
        int[] edades = { 17, 19, 18, 18, 18 };
        
        for (int i = 0; i < edades.length; i++) {
            System.out.println("indice: " + i + " es: " + edades[i]);
            
            if (edades[i] >= 18)
                System.out.println("Eres Mayor");
            else
                System.out.println("Eres Menor");
        }
    }
}

